cli/src/commands/extensions/extension.rs

```diff
@@ -124,8 +125,18 @@ impl Extension {
 
     /// Execute an extension subcommand.
     pub async fn run(&self, api: BoxFuture<'static, Result<PhylumApi>>) -> CommandResult {
+        let args = env::args()
+            .skip_while(|arg| arg != self.name())
+            .skip(1)
+            .collect();
```

Comment:
This approach is a little unfortunate, theoretically this means that an extension with the name `phylum` will get all arguments and other similar conflicts with existing parameters. But I'm not sure we have much better options?

Reply:
The issue of an extension named `phylum` can be resolved with an extra `.skip(1)`. But that doesn't solve other problems...

```suggestion
        let args = env::args()
            .skip(1)
            .skip_while(|arg| arg != self.name())
            .skip(1)
            .collect();
```

Reply:
> The issue of an extension named phylum can be resolved with an extra .skip(1). But that doesn't solve other problems...

Yeah, was considering to mention that but it's really irrelevant. The problem persists either way.